src/tools/recvfuzz/recvfuzz.cpp

```diff
@@ -505,25 +683,222 @@ void buildInitialPacket(CXPLAT_SOCKET* Binding, CXPLAT_ROUTE Route, int64_t* Pac
     }
 }
 
-void fuzzHandshakePacket() {
-// TODO
-}
-
 void fuzz(CXPLAT_SOCKET* Binding, CXPLAT_ROUTE Route) {
     int64_t PacketCount = 0;
     int64_t TotalByteCount = 0;
-    uint8_t mode;
+    uint8_t mode = 1;
     uint64_t StartTimeMs = CxPlatTimeMs64();
+    uint64_t SrcCid;
+    PacketParams HandshakePacketParams = {
+        sizeof(uint64_t),
+        sizeof(uint64_t),
+        nullptr,
+        nullptr,
+        0,
+        1
+    };
+    CxPlatEventInitialize(&(RecvPacketEvent), TRUE, FALSE);
```

Comment:
This info should be passed into the constructor of `RecvPacketEvent`, and not called directly.